src/utils.js

```diff
@@ -147,3 +147,17 @@ export const verifyJWT = (token) => {
     });
   });
 };
+
+/*
+Get JSON Web Token
+*/
+export const getToken = (req, res) => {
+  var secret = 'secretkey';
```

Comment:
This secret is already defined above in the variable `TOKEN_SIGNATURE`. Moving the token value to a config should solve this issue of having to define and maintain it in multiple places

Reply:
I pushed some new changes to dev that now have a config file with this value. Please import it from there